useragent/mac/DNSFilter_Debugger_mac.sh

```diff
@@ -1,26 +1,27 @@
 #!/bin/bash
-echo "======================================================" >> $ResultLog
+
+# Store all results to this file
+ResultLog=$HOME/Downloads/DNSFilterLogs/DNSFilterDebug.txt
+echo "" > $ResultLog
+
+echo "======================================================" 
 echo "DNSFilter Debugger for Mac"
 echo "The purpose of this tool is to assist support engineers with common requests."
 #CREATED BY RICK COHEN DEC 2022
-echo "======================================================" >> $ResultLog
+echo "======================================================" 
 # Create the "DNSFilter Logs" directory if it doesn't exist
 if [ ! -d "$HOME/Downloads/DNSFilterLogs" ]; then
-  mkdir $HOME/Downloads/DNSFilterLogs
+  mkdir -p $HOME/Downloads/DNSFilterLogs
 fi
```

Comment:
Move this up before the variable declaration since you are doing that echo statement